include/wil/rpc_helpers.h

```diff
@@ -132,6 +131,43 @@ namespace wil
         RpcEndExcept
     }
 
+    namespace details
+    {
+        // Provides an adapter around calling the context-handle-close method on an
+        // RPC interface, which itself is an RPC call.
+        template<typename TStorage, typename close_fn_t, close_fn_t close_fn>
```

Comment:
Will the `TStorage` always be a trivially-copyable thing no bigger than a pointer?

Reply:
Pretty sure, RPC handles must be pointer-sized.